src/context.ts

```diff
@@ -0,0 +1,109 @@
+import { grpc } from '@improbable-eng/grpc-web'
+import { ThreadID } from '@textile/threads-id'
+import { Config } from '@textile/threads-client'
+
+type HostString = 'https://hub.textile.io:443' | 'https://hub.staging.textile.io:443' | 'http://127.0.0.1:3007' | string
```

Comment:
since it can just be a `string` what is the point of the other options that are just strings?

Reply:
Autocomplete :)

Reply:
(also mostly just reminders for myself)

Reply:
You shouldn't need the port on the hub ones, right? I can't remember, is the proxy on 447?